src/modules/shared/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import { Sidebar, Menu, MenuItem } from "react-pro-sidebar";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+export default function SideBar() {
+  const [isCollapse, setIsCollapse] = useState(true);
+  const toggleCollapse = () => {
+    setIsCollapse(!isCollapse);
+  };
+  return (
+    <>
+      <div className="sidebar-container">
+        <Sidebar collapsed={isCollapse} className="rounded-R">
+          <span
+            className="toggle-icon  d-flex align-items-center justify-content-center"
+            onClick={toggleCollapse}
+          >
+            <i
+              className={`fa-solid text-white ${
+                isCollapse ? "fa-chevron-right" : "fa-chevron-left"
+              }`}
+            ></i>
+          </span>
+          <Menu className="vh-100 mt-5">
+            <MenuItem
+              icon={<div className="fa fa-home mx-3 menu-item"></div>}
+              component={<Link to="/Dashboard" />}
+            >
+              Home{" "}
+            </MenuItem>
+            <MenuItem
+              icon={
+                <div className="fa-solid fa-user-group mx-3 menu-item"></div>
+              }
+              component={<Link to="/usres" />}
+            >
+              Users
+            </MenuItem>
+            <MenuItem
+              icon={
+                <svg
```

Comment:
We can make the icon as a  component in the assets folder and re-use it here.  We can use this website as well to convert svg to jsx
https://svg2jsx.com/

Reply:
done hope you review the new code